### ISSUE TITLE
Add a license?

### ISSUE THREAD

**Issue by emmyft** [SaraVilla/OLS-project-sequencing-tutorial#1]
This looks really exciting! To ensure others can share and reuse your work (and work in this project in the future) - might be nice to add an open license, maybe [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)? 🌟

**Comment by SaraVilla** [comment]
Yes, definitely!! I still have to add the Licence and Code of Conduct (some ticks missing form the OLS checklist ;) )
thanks!
